.github/workflows/testBuild.yml

```diff
@@ -152,7 +152,15 @@ jobs:
         run: npm run build-adhoc
 
       - name: Deploy to S3 for internal testing
-        run: aws s3 cp --recursive --acl public-read "$GITHUB_WORKSPACE"/dist s3://ad-hoc-expensify-cash/web/"$PULL_REQUEST_NUMBER"
+        run: aws s3 cp --recursive --acl public-read "$GITHUB_WORKSPACE/dist" s3://ad-hoc-expensify-cash/web/${{ env.PULL_REQUEST_NUMBER }}
+
+      - name: Setup Cloudflare CLI
+        run: pip3 install cloudflare==2.19.0
+
+      - name: Purge Cloudflare cache
+        run: /home/runner/.local/bin/cli4 --verbose --delete hosts=["ad-hoc-expensify-cash/web/${{ env.PULL_REQUEST_NUMBER }}"] /zones/:9ee042e6cfc7fd45e74aa7d2f78d617b/purge_cache
```

Comment:
The host part here isn't valid. It's still going to be `ad-hoc-expensify-cash` and everything after that is the path. I assume there is some option in this CLI to add a path match on it too

Reply:
Actually I think what you want here is this

`hosts=["<pr_number>.pr-testing.expensify.com"]`

And it only needs to run if this is the second build in the same PR if that is simple enough to figure out